lib/src/Uncertainty/StatTests/VisualTest.cxx

```diff
@@ -73,14 +72,14 @@ Graph VisualTest::DrawQQplot(const Sample & sample,
 {
   if (sample.getDimension() != 1) throw InvalidDimensionException(HERE) << "Error: can draw a QQplot only if dimension equals 1, here dimension=" << sample.getDimension();
   if (dist.getDimension() != 1) throw InvalidDimensionException(HERE) << "Error: can draw a QQplot only if dimension equals 1, here dimension=" << dist.getDimension();
-  const UnsignedInteger size = sample.getSize();
-  const Sample sortedSample(sample.sort(0));
+  const Sample sortedSample(sample.sortUnique());
+  const UnsignedInteger size = sortedSample.getSize();
   Sample data(size, 2);
-  const Scalar step = 1.0 / size;
   for (UnsignedInteger i = 0; i < size; ++i)
   {
     data(i, 0) = sortedSample(i, 0);
-    data(i, 1) = dist.computeQuantile((i + 0.5) * step)[0];
+    const Scalar p = sample.computeEmpiricalCDF(sortedSample[i]);
+    data(i, 1) = dist.computeQuantile(p)[0];
```

Comment:
when i==size-1, we have p==1.0 and dist.computeQuantile(p)[0] will be (most of the time) much larger than sample.getMax(), resulting into a very distorted graph. @adutfoy suggested to remove the last point from the graph, which looks like a good idea, at least better than mine when I introduced this +0.5 shift!